Document staticPageGenerationTimeout config (#32306)
vercel/next.js: docs/basic-features/data-fetching.md

| Version   | Changes                                                                                                           |
| --------- | ----------------------------------------------------------------------------------------------------------------- |

> | `v12.0.0` | Default static generation timeout added                                                                           |

Reviewer: ```suggestion
| `v12.0.0` | `staticPageGenerationTimeout` added                                                                            |
```

export default Blog
```

> Note: by default Next.js has a timeout for static generation where if no activity occurs for 60 seconds it will re-attempt generation three times and if it still fails after retrying the build will fail. This timeout can be increased if needed via the below config:

Reviewer: ```suggestion
Note: Next.js has a default timeout for static generation where if no activity occurs for 60 seconds it will re-attempt generation three times. If the fourth attempt fails, the build will fail. This timeout can be increased if needed using the below configuration:
```

export default Blog
```

> Note: by default Next.js has a timeout for static generation where if no new pages complete generating within 60 seconds it will re-attempt generation three times and if it still fails after retrying the build will fail. This timeout can be increased if needed via the below config:

Reviewer: ```suggestion
Note: Next.js has a default static generation timeout of 60 seconds. If no new pages complete generating within the timeout period, it will attempt generation three more times. If the fourth attempt fails, the build will fail. This timeout can be modified using the following configuration:
```
